repos/system_upgrade/el7toel8/actors/repositoriesmapping/libraries/library.py

```diff
@@ -15,6 +15,13 @@ def inhibit_upgrade(title):
     raise StopActorExecution()
 
 
+def strip_comments(file_):
+    for line in file_:
+        data = line.split('#')[0].strip()
```

Comment:
What about if we ignore a line starting with "#" and communicate that only new line comments are valid?